aws/table_aws_eks_identity_provider_config.go

```diff
@@ -0,0 +1,228 @@
+			Hydrate:    getEksIdentityProviderConfig,
+		},
+		List: &plugin.ListConfig{
+			ParentHydrate: listEksClusters,
+			Hydrate:       listEksIdentityProviderConfig,
+		},
+		GetMatrixItem: BuildRegionList,
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The name of the identity provider configuration.",
+				Type:        proto.ColumnType_STRING,
+				// Transform:   transform.FromP(getAwsEksProviderConfigNameAndType, "Name"),
+			},
+			{
+				Name:        "type",
+				Description: "The type of the identity provider configuration.",
+				Type:        proto.ColumnType_STRING,
+				// Transform:   transform.FromP(getAwsEksProviderConfigNameAndType, "Type"),
+			},
+			{
+				Name:        "cluster_name",
+				Description: "The name of the cluster.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "arn",
+				Description: "The Amazon Resource Name (ARN) of the configuration.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("IdentityProviderConfigArn"),
+				Hydrate:     getEksIdentityProviderConfig,
+			},
+			{
+				Name:        "groups_claim",
+				Description: "The JSON web token (JWT) claim that the provider uses to return your groups.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getEksIdentityProviderConfig,
+			},
+			{
+				Name:        "groups_prefix",
+				Description: "The prefix that is prepended to group claims to prevent clashes with existing names (such as system: groups).",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getEksIdentityProviderConfig,
+			},
+			{
+				Name:        "issuer_url",
+				Description: "The URL of the OIDC identity provider that allows the API server to discover public signing keys for verifying tokens.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getEksIdentityProviderConfig,
+			},
+			{
+				Name:        "required_claims",
+				Description: "The key-value pairs that describe required claims in the identity token.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getEksIdentityProviderConfig,
+			},
+			{
+				Name:        "status",
+				Description: "The status of the OIDC identity provider.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getEksIdentityProviderConfig,
+			},
+			{
+				Name:        "tags_src",
+				Description: "The metadata to apply to the provider configuration to assist with categorization and organization.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getEksIdentityProviderConfig,
+				Transform:   transform.FromField("Tags"),
+			},
+			{
+				Name:        "username_claim",
+				Description: "The JSON Web token (JWT) claim that is used as the username.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getEksIdentityProviderConfig,
+			},
+			{
+				Name:        "username_prefix",
+				Description: "The JSON Web token (JWT) claim that is used as the username.",
```

Comment:
Ordering of columns is not correct, JSON columns should come after the string columns.

Reply:
Managed the column ordering